tests/openedx_tagging/core/tagging/test_api.py

```diff
@@ -622,8 +744,8 @@ def get_object_tags():
 
         # Now retrieve the tags again:
         assert get_object_tags() == [
-            "Languages: English",
-            "Life on Earth: Eukaryota>Animalia>Chordata",
+            "-1-languages: English",
```

Comment:
While we're at it, can we change the default export_id of the Languages taxonomy to just `languages` ?

Reply:
@bradenmacdonald Do you mean changing it using migrations? or only in this test?

Reply:
I meant using a migration. `-1-languages` is not a good ID, and because it's currently the default it's going to show up in a lot of export files and stuff in the future. I'd rather change it to `languages` or `languages-v1` or something nicer. It's not a big deal though.

Reply:
Updated here d966b1359fdfb51f9066f92aaac7ab9d9f6e24ec

Reply:
Updated here d966b1359fdfb51f9066f92aaac7ab9d9f6e24ec

Reply:
Thanks!